Configurations/unix-Makefile.tmpl

```diff
@@ -323,6 +323,9 @@ install_sw: all install_dev install_engines install_runtime
 
 uninstall_sw: uninstall_runtime uninstall_engines uninstall_dev
 
+doc-nits:
+	$(PERL) util/find-doc-nits.pl -n
+
```

Comment:
I think this belongs in the developer targets further down (search for "Developer targets")